src/Hng.Application/Features/Languages/Dtos/LanguagesListResponseDto.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace Hng.Application.Features.Languages.Dtos
+{
+    public class LanguagesListResponseDto
+    {
+        [JsonPropertyName("languages")]
+        public List<LanguageDto> Languages { get; set; }
```

Comment:
Rename this field to data so as to follow convention

Reply:
I do not understand what you mean here as well